packages/create-svelte/shared/+playwright+typescript/playwright.config.ts

```diff
@@ -4,7 +4,8 @@ const config: PlaywrightTestConfig = {
 	webServer: {
 		command: 'npm run build && npm run preview',
 		port: 4173
-	}
+	},
+	testMatch: 'tests/**/*.{js,ts}'
```

Comment:
I think adding this here as Rich suggested is a decent idea, but I might tweak it a bit. By default playwright looks for `test` or `spec` in the name and it might not be a bad idea to keep their default setting, but just add the `tests/` directory prefix so that we don't try to run non-test files
```suggestion
	testMatch: 'tests/**/.*(test|spec)\.(js|ts|mjs)'
```

Reply:
But the Playwright runner is going to conflict with the VitTest runner. You can't keep both Playwright default and Vitest default settings if the goal is to have Vitest runner only run its test and Playwright run only its test. Or are we having a preference for one default runner? 

I choose to have each runner with it's own files bc the Playwright runner has extra functionality but it required separating the Playwright files to have .e2e  to avoid changing Vitest default settings. `testMatch: 'tests/e2e/**.e2e.js'` for playwright.

The other option would be to make test/unit folder and test/e2e folder and change both the testMatch for both Playwright and ViTest to read only their folders. Could also add an integration/component folder but it doesn't seem if a decision on whether to use testing-library with svelte or Playwright has been made there.

Reply:
Rich and I were thinking that you'd differentiate by directory. So the Playwright tests would be in `tests/` and the Vitest tests would be in `src/`. I don't know if that's the most common way of doing it or not, but that's what we've been doing in SvelteKit's own tests and it's be working pretty well there